assignment/assignment.hs

```diff
@@ -52,8 +58,12 @@ count element list = 0 -- replace me
 -- help to write it as two functions. (Hint: you only
 -- need to traverse the list once)
 
-evensEqualOdds :: [ Integer ] -> Bool
-evensEqualOdds numbers = False -- replace me
+evensEqualOdds :: [ Int ] -> Bool
+evensEqualOdds numbers = if(alternateSumList numbers == 0) then True else False
+
```

Comment:
Hah!

A lot of professional programmers do this, too, so don't feel bad. But:

~~~ hs
if test-expression then True else False
~~~

is exactly the same as writing

~~~
test-expression
~~~

:)